ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/structured_configs/DC1-BL1B.yml

```diff
@@ -531,6 +537,49 @@ vlans:
 - id: 350
   name: Tenant_C_WAN_Zone_1
   tenant: Tenant_C
+ip_access_lists:
+- name: TEST-IPV4-ACL-WITH-IP-FIELDS-IN_Ethernet8
+  entries:
+  - sequence: 15
+    action: deny
+    protocol: ip
+    source: any
+    destination: 10.10.10.10
+- name: TEST-IPV4-ACL-WITH-IP-FIELDS-IN_Ethernet9
+  entries:
+  - sequence: 15
+    action: deny
+    protocol: ip
+    source: any
+    destination: 10.10.20.20
+- name: TEST-IPV4-ACL-WITH-IP-FIELDS-IN_Ethernet10
+  entries:
+  - sequence: 15
+    action: deny
+    protocol: ip
+    source: any
+    destination: 10.10.30.10
+- name: TEST-IPV4-ACL-WITH-IP-FIELDS-OUT_Ethernet8
+  entries:
+  - remark: Some remark will not require source and destination fields.
+  - action: permit
+    protocol: ip
+    source: 10.10.10.10
+    destination: any
+- name: TEST-IPV4-ACL-WITH-IP-FIELDS-OUT_Ethernet9
+  entries:
+  - remark: Some remark will not require source and destination fields.
+  - action: permit
+    protocol: ip
+    source: 10.10.20.20
+    destination: any
+- name: TEST-IPV4-ACL-WITH-IP-FIELDS-OUT_Ethernet10
+  entries:
+  - remark: Some remark will not require source and destination fields.
+  - action: permit
+    protocol: ip
+    source: 10.10.30.10
```

Comment:
This seems to be the wrong IP for DC1-BL1B on Ethernet10, so I think this is proving my point from the other comment, that we should evaluate the `nodes` key and skip interfaces that are not for us.

Reply:
Fixed this problem. Now it is rendering the acl for interfaces of DC1-BL1B  only.